content/influxdb/v2.1/notebooks/clean-data.md

```diff
@@ -10,31 +10,154 @@ menu:
+        else if d >= 123.75 and d < 146.25 then "SE"
+        else if d >= 146.25 and d < 168.75 then "SSE"
+        else if d >= 168.75 and d < 191.25 then "S"
+        else if d >= 191.25 and d < 213.75 then "SSW"
+        else if d >= 213.75 and d < 236.25 then "SW"
+        else if d >= 236.25 and d < 258.75 then "WSW"
+        else if d >= 258.75 and d < 281.25 then "W"
+        else if d >= 281.25 and d < 303.75 then "WNW"
+        else if d >= 303.75 and d < 326.25 then "NW"
+        else if d >= 326.25 and d < 348.75 then "NNW"
+        else ""
+
+        return _cardinal
+    }
+
+    __PREVIOUS_RESULT__
+        |> map(fn: (r) => ({r with
+            _field: "wind_dir_cardinal",
+            _value: cardinalDir(d: r._value),
+        }))
+    ```
+    {{% /oss-only %}}
+
+    {{% cloud-only %}}
+
+    ```js
+    import "array"
+
+    cardinalDir = (d) => {
+        _cardinal = if d >= 348.7 or d < 11.25 then "N"
+        else if d >= 11.25 and d < 33.75 then "NNE"
+        else if d >= 33.75 and d < 56.25 then "NE"
+        else if d >= 56.25 and d < 78.75 then "ENE"
+        else if d >= 78.75 and d < 101.25 then "E"
+        else if d >= 101.25 and d < 123.75 then "ESE"
+        else if d >= 123.75 and d < 146.25 then "SE"
+        else if d >= 146.25 and d < 168.75 then "SSE"
+        else if d >= 168.75 and d < 191.25 then "S"
+        else if d >= 191.25 and d < 213.75 then "SSW"
+        else if d >= 213.75 and d < 236.25 then "SW"
+        else if d >= 236.25 and d < 258.75 then "WSW"
+        else if d >= 258.75 and d < 281.25 then "W"
+        else if d >= 281.25 and d < 303.75 then "WNW"
+        else if d >= 303.75 and d < 326.25 then "NW"
+        else if d >= 326.25 and d < 348.75 then "NNW"
+        else ""
+
+        return _cardinal
+    }
+
+    __PREVIOUS_RESULT__
+        |> map(fn: (r) => ({r with
+            _field: "wind_dir_cardinal",
+            _value: cardinalDir(d: r._value),
+        }))
+        |> to(bucket: "noaa")
+    ```
+    {{% /cloud-only %}}
+
+4.  {{% oss-only %}}
+    
+    Click {{% icon "notebook-add-cell" %}} after your **Flux Script** cell to 
+    add a new cell and select **{{% caps %}}Output to Bucket{{% /caps %}}**.
+    Select your **noaa** bucket from the **{{% icon "bucket" %}} Choose a bucket**
+    drop-down menu. 
+
+    {{% /oss-only %}}
+    
+5.  _(Optional)_ Click {{% icon "notebook-add-cell" %}} and select **Note** to
+    add a cell containing notes about what this notebook does. For example, the
+    cell might say, "This notebook converts decimal degree wind direction values
+    to cardinal directions."
+6.  {{% oss-only %}}
+
+    Click **Preview** in the upper left to verify that your notebook runs and previews the output.
+    
+    {{% /oss-only %}}
+6.  Click **Run** to run the notebook and write the normalized data back to the
+    **noaa** bucket.
+
+## Continuously run a notebook
+To write continuously run your notebook, normalize data, and write it to a bucket,
```

Comment:
```suggestion
To write continuously, run your notebook, normalize data, and write it to a bucket,
```

Reply:
for this particular example, think running the notebook covers normalizing data and writing to a bucket? if yes, perhaps:

To write continuously run your notebook, export the notebook as a task: